editoast/fga/src/client/tuples.rs

```diff
@@ -24,6 +27,66 @@ impl<'a, R: Relation, U: AsUser<User = R::User>> From<&Tuple<'a, R, U>> for RawT
 }
 
 impl Client {
+    #[tracing::instrument(skip(self), err)]
+    pub(super) async fn get_stores_read<'a>(
+        &self,
+        store_id: &str,
+        tuple_key: RawTuple,
+        page_size: Option<usize>,
+        authorization_model_id: Option<&str>,
+        consistency: Option<Consistency>,
+    ) -> Result<(Vec<RawTuple>, String), RequestFailure> {
+        #[derive(serde::Serialize)]
+        struct Request<'a> {
+            tuple_key: RawTuple,
+            #[serde(skip_serializing_if = "Option::is_none")]
+            page_size: Option<usize>,
+            #[serde(skip_serializing_if = "Option::is_none")]
+            authorization_model_id: Option<&'a str>,
+            #[serde(skip_serializing_if = "Option::is_none")]
+            consistency: Option<Consistency>,
+        }
+
+        let url = self
+            .base_url()
+            .join(format!("stores/{store_id}/read").as_str())
+            .unwrap();
```

Comment:
nitpick: I would rather describe all the `struct` first, then the actual code after.

Reply:
done https://github.com/OpenRailAssociation/osrd/compare/97317ee8401ed2f2c1cea3154bbac98e8c7f7fe6..c55f8ca1a15cb7a06260088595446de7a225aebe